docs: Update website README.md with .env instructions (#6501)
CloudFire-LLC/cloudfire: website/README.md

pnpm setup
```

> Next, create files `.env.local` and `.env.development.local` in the root folder. 

Reviewer: ```suggestion
Next, create files `.env.local` and `.env.development.local` in this directory. 
```
